open-banking-accelerator/components/consent-management/com.wso2.openbanking.accelerator.consent.extensions/src/test/java/com/wso2/openbanking/accelerator/consent/extensions/manage/vrp/VRPConsentHandlerTest.java

```diff
@@ -0,0 +1,217 @@
+package com.wso2.openbanking.accelerator.consent.extensions.manage.vrp;
+
+
+import com.wso2.openbanking.accelerator.common.config.OpenBankingConfigParser;
+import com.wso2.openbanking.accelerator.common.exception.ConsentManagementException;
+import com.wso2.openbanking.accelerator.common.util.CarbonUtils;
+import com.wso2.openbanking.accelerator.consent.extensions.common.ConsentException;
+import com.wso2.openbanking.accelerator.consent.extensions.common.ConsentServiceUtil;
+import com.wso2.openbanking.accelerator.consent.extensions.internal.ConsentExtensionsDataHolder;
+import com.wso2.openbanking.accelerator.consent.extensions.manage.impl.ConsentManageRequestHandler;
+import com.wso2.openbanking.accelerator.consent.extensions.manage.impl.VRPConsentRequestHandler;
+import com.wso2.openbanking.accelerator.consent.extensions.manage.model.ConsentManageData;
+import com.wso2.openbanking.accelerator.consent.extensions.utils.ConsentExtensionTestUtils;
+import com.wso2.openbanking.accelerator.consent.mgt.dao.models.ConsentResource;
+import com.wso2.openbanking.accelerator.consent.mgt.service.ConsentCoreService;
+import com.wso2.openbanking.accelerator.consent.mgt.service.impl.ConsentCoreServiceImpl;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.testng.PowerMockTestCase;
+import org.testng.IObjectFactory;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.ObjectFactory;
+import org.testng.annotations.Test;
+
+import java.time.OffsetDateTime;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+import static org.mockito.Matchers.anyBoolean;
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Mockito.mock;
+import static org.powermock.api.mockito.PowerMockito.doReturn;
+import static org.powermock.api.mockito.PowerMockito.when;
+
+/**
+ * comment.
+ */
+@PowerMockIgnore({"jdk.internal.reflect.*",
+"com.wso2.openbanking.accelerator.consent.extensions.common.*"})
+@PrepareForTest({OpenBankingConfigParser.class, ConsentServiceUtil.class,
+        ConsentExtensionsDataHolder.class})
+public class VRPConsentHandlerTest extends PowerMockTestCase {
+
+    @Mock
+    ConsentManageData consentManageDataMock;
+
+    ConsentManageRequestHandler consentManageRequestHandler;
+    public static final String DOMESTIC_VRP_CONSENT_PATH = "domestic-vrp-consents";
+
+    private static final String INVALID_VRP_PATH = "domestic-vrp-consent/34567890987";
+
+    public static final OffsetDateTime EXPIRATION_DATE = OffsetDateTime.now().plusDays(50);
+    public static final OffsetDateTime TRANSACTION_FROM_DATE = OffsetDateTime.now();
```

Comment:
Shall we move these constants to a common place? Also check whether these constants are already defined in a common place